File: JavaScript/Final Value of Variable After Performing Operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
    let num = 0;
    for (i=0; i<operations.length;i++){

if(operations[i].includes("--")){
    num = num -1;
}
if(operations[i].includes("++")){
    num = num +1
}


    }
    return num
};

File: Python/final_value_of_variable_after_performing_operations.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        num = 0
        for i in range(0,len(operations)):
            if "--" in operations[i]:
                num = num-1
            elif "++" in operations[i]:
                num = num+1
        return num
        
        
